manual_tests/internal/internalize.go

```diff
@@ -0,0 +1,114 @@
+package internal
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/bsv-blockchain/go-sdk/chainhash"
+	"github.com/bsv-blockchain/go-sdk/script"
+	"github.com/bsv-blockchain/go-sdk/transaction"
+	"github.com/bsv-blockchain/go-sdk/transaction/template/p2pkh"
+	sdk "github.com/bsv-blockchain/go-sdk/wallet"
+	"github.com/bsv-blockchain/go-wallet-toolbox-manual-tests/internal/fixtures"
+	"github.com/bsv-blockchain/go-wallet-toolbox/pkg/brc29"
+	"github.com/go-softwarelab/common/pkg/slices"
+)
+
+func (m *Manager) InternalizeTxID(txID string, user fixtures.UserConfig, keyID brc29.KeyID, address string) (fixtures.Summary, error) {
+	var summary fixtures.Summary
+
+	txIDHash, err := chainhash.NewHashFromHex(txID)
+	if err != nil {
+		return summary, fmt.Errorf("invalid txID %q: %w", txID, err)
+	}
+
+	summary = append(summary, fmt.Sprintf("Fetching atomic beef for txID %q on network %q", txID, m.GetBSVNetwork()))
+
+	beef, err := m.storageInfra.Services.GetBEEF(m.ctx, txID, nil)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get beef for txID %q, network %q: %w", txID, m.GetBSVNetwork(), err)
```

Comment:
```suggestion
		return summary, fmt.Errorf("failed to get beef for txID %q, network %q: %w", txID, m.GetBSVNetwork(), err)
```
why do not return summary here if everywhere else u do ?